client/src/components/Onboarding/Onboarding.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import { Route, Switch, Redirect, useRouteMatch } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { BrowserChecker } from '_common';
+import Sidebar from '../Sidebar';
+import OnboardingAdmin from './OnboardingAdmin';
+import OnboardingUser from './OnboardingUser';
+
+function Onboarding() {
+  const dispatch = useDispatch();
+  const { path } = useRouteMatch();
+
+  useEffect(() => {
+    dispatch({ type: 'FETCH_AUTHENTICATED_USER' });
+    dispatch({ type: 'FETCH_NOTIFICATIONS' });
+  }, []);
+
+  return (
+    <div className="workbench-wrapper">
```

Comment:
__Question__: Why recreate the Workbench (again, like Manage Account) instead of using the Workbench component to render a section, but __not__ render a link to it?

Reply:
We're going to move the /onboarding route under /workbench/onboarding.  So we won't be recreating the workbench then. I'll leave this unresolved until that step is done.

Reply:
@tacc-wbomar , has been done in a recent commit. so resolving this.